### PR TITLE
Event page filter negation

### PR DESCRIPTION
**Pull request by jgoley**
Here's an initial attempt at establishing the UX and logic for 3 state filtering on the events page.
We probably need to add a simple explanation somehow—I'll do that tomorrow.

[PT story](https://www.pivotaltracker.com/story/show/150920530)

### CODE REVIEW

File: client/controllers/events/eventFiltration.coffee [hunk 1]
```diff
@@ -6,6 +6,42 @@ formatDateForInput = (date) ->
     return moment()
   if date.getTime then date else new moment(date)
 
+# Update input classes based on state of button: unselected, positive, negative
+updateInputState = (event) ->
```
Comment (nathanathan) [inline client/controllers/events/eventFiltration.coffee]: It seems like this should be handled by the jade template.

Comment (jgoley) [reply client/controllers/events/eventFiltration.coffee]: Good point. I've been working on a refactor that reduces storing the state of the buttons in the DOM. It takes care of this issue but currently running into an issue with conflicting events. I'll push an update ASAP.

**Comment by jgoley** [comment]
@nathanathan I think this is ready for final review.

**Comment by nathanathan** [comment]
👍

**Pull request merged**